packages/p2p/src/components/app.jsx

```diff
@@ -67,6 +67,11 @@ const App = props => {
             general_store.redirectTo('orders');
             order_store.setOrderId(order_id);
         }
+        general_store.props.setP2POrderProps({
+            order_id,
+            redirectToOrderDetails: general_store.redirectToOrderDetails,
+            setIsRatingModalOpen: order_store.setIsRatingModalOpen,
+        });
```

Comment:
called setP2POrderProps whenever order_id changes, so when clicking between the different order give feedback buttons, the user can navigate to each order. If this is not here, there will be a bug where the user could navigate from one order to the other, but then wouldn't be able to click the give feedback button as the order_id in p2p_order_props won't be updated

<img width="298" alt="Screenshot 2022-09-21 at 4 58 15 PM" src="https://user-images.githubusercontent.com/103412909/191461910-f8a7ebc1-73c6-4624-b39f-3a7235c02adb.png">